### PR TITLE
Fix jline lib issue during bundle:start.

### PR DESCRIPTION
**Pull request by Darkvar**
Fixing an error during plugin [temporary install](https://github.com/sonatype-nexus-community/nexus-repository-helm#temporary-install).
```
Error executing command: Error executing command on bundles:
        Error starting bundle 251: Unable to resolve org.sonatype.nexus.plugins.nexus-repository-helm [251](R 251.0): missing requirement [org.sonatype.nexus.plugins.nexus-repository-helm [251](R 251.0)] osgi.wiring.package; (&(osgi.wiring.package=jline.internal)(version>=2.14.0)) Unresolved requirements: [[org.sonatype.nexus.plugins.nexus-repository-helm [251](R 251.0)] osgi.wiring.package; (&(osgi.wiring.package=jline.internal)(version>=2.14.0))]
```

This pull request makes the following changes:
* replacing `jline.internal.Preconditions.checkNotNull` with `java.util.Objects.requireNonNull`

### CODE REVIEW

**Comment by Darkvar** [comment]
@DarthHater could you please take a look

File: src/main/java/org/sonatype/repository/helm/internal/hosted/HostedHandlers.java [hunk 1]
```diff
@@ -24,7 +24,7 @@
 import org.sonatype.repository.helm.internal.AssetKind;
 import org.sonatype.repository.helm.internal.util.HelmPathUtils;
 
-import static jline.internal.Preconditions.checkNotNull;
+import java.util.Objects;
```
Comment (bhamail) [inline src/main/java/org/sonatype/repository/helm/internal/hosted/HostedHandlers.java]: not a big deal, but in other places in the codebase, we've used:

    import static com.google.common.base.Preconditions.checkNotNull;

might want to use that for consistency.

Whatever you decide, you may want to resort the imports.

Comment (DarthHater) [reply src/main/java/org/sonatype/repository/helm/internal/hosted/HostedHandlers.java]: +1 to that

File: src/main/java/org/sonatype/repository/helm/internal/hosted/HostedHandlers.java [hunk 2]
```diff
@@ -42,7 +42,7 @@
 
   @Inject
   public HostedHandlers(final HelmPathUtils helmPathUtils) {
-    this.helmPathUtils = checkNotNull(helmPathUtils);
+    this.helmPathUtils = Objects.requireNonNull(helmPathUtils);
```
Comment (DarthHater) [inline src/main/java/org/sonatype/repository/helm/internal/hosted/HostedHandlers.java]: I would switch to using `import static com.google.common.base.Preconditions.checkNotNull;` as @bhamail pointed out, that's consistent with Nexus Repo

File: .travis.yml [hunk 3]
```diff
@@ -1 +1,11 @@
 language: java
+
```
Comment (DarthHater) [inline .travis.yml]: These changes look unrelated to your fix, why did you need to add these changes to Travis?

File: .travis.yml [hunk 4]
```diff
@@ -1 +1,11 @@
...
+dist: trusty
+
+jdk:
+  - oraclejdk8
```
Comment (DarthHater) [inline .travis.yml]: Pretty sure you could use openjdk8 rather than Oracle, as the rest of our stuff seems to be using that now.

File: .travis.yml (hunk 3 above)
Comment (Darkvar) [reply .travis.yml]: @DarthHater Travis is using `java11` by default for now and I'm forcing it to use `java8`

**Comment by Darkvar** [comment]
Closing this PR in favour #55
